src/templates/base.html.twig

```diff
@@ -339,12 +353,21 @@
                                 <ul class="notika-main-menu-dropdown">
                                 </ul>
                             </div>
+                            
                             <div id="projects" class="tab-pane {% if app.request.attributes.get( '_route' ) starts with 'project_' %}in active{% endif %} notika-tab-menu-bg animated flipInX">
                                 <ul class="notika-main-menu-dropdown">
                                     {{ render(controller('App\\Controller\\NavigationController::projectList')) }}
                                 </ul>
                             </div>
 
+                            {% if project is defined %}
+                                <div id="iterations" class="tab-pane {% if app.request.attributes.get( '_route' ) starts with 'iteration_' %}in active{% endif %} notika-tab-menu-bg animated flipInX">
```

Comment:
Should `in active` be `inactive`?

Reply:
Looks like `in` doesn't belong at all.  I was just following what the template was doing.  I removed `in` from the two menu items above and tested and everything still works as I would expect.